metadata-service/graphql-servlet-impl/src/main/java/com/datahub/graphql/GraphQLController.java

```diff
@@ -105,59 +106,59 @@ CompletableFuture<ResponseEntity<String>> postGraphQL(HttpEntity<String> httpEnt
                 .convertValue(variablesJson, new TypeReference<Map<String, Object>>() {})
             : Collections.emptyMap();
 
-    log.debug(String.format("Executing graphQL query: %s, variables: %s", queryJson, variables));
-
     /*
      * Init QueryContext
      */
     Authentication authentication = AuthenticationContext.getAuthentication();
 
     SpringQueryContext context =
         new SpringQueryContext(
-            true,
-            authentication,
-            _authorizerChain,
-            systemOperationContext,
-            queryJson.asText(),
-            variables);
+            true, authentication, _authorizerChain, systemOperationContext, query, variables);
     Span.current().setAttribute("actor.urn", context.getActorUrn());
 
+    final String threadName = Thread.currentThread().getName();
+    log.info(
+        "Processing request, operation: {}, actor urn: {}", operationName, context.getActorUrn());
+    log.debug("Query: {}, variables: {}", query, variables);
+
     return CompletableFuture.supplyAsync(
         () -> {
+          log.info("Executing operation {} for {}", operationName, threadName);
+
           /*
            * Execute GraphQL Query
            */
           ExecutionResult executionResult =
-              _engine.execute(queryJson.asText(), operationName, variables, context);
+              _engine.execute(query, operationName, variables, context);
 
           if (executionResult.getErrors().size() != 0) {
             // There were GraphQL errors. Report in error logs.
             log.error(
-                String.format(
-                    "Errors while executing graphQL query: %s, result: %s, errors: %s",
-                    queryJson, executionResult.toSpecification(), executionResult.getErrors()));
-          } else {
-            log.debug(
-                String.format(
-                    "Executed graphQL query: %s, result: %s",
-                    queryJson, executionResult.toSpecification()));
+                "Errors while executing query: {}, result: {}, errors: {}",
+                query,
```

Comment:
You probably want to truncate the query, they can be very large even though they are requests.